frontend/shared/smartTags/SmartTagEditor.js

```diff
@@ -5,6 +5,10 @@ import SmartTagContainer from "./SmartTagContainer";
 class SmartTagEditor extends SmartTagContainer {
     constructor($el, options) {
         $el.quillify();
+        if (options.takeFocusEl) {
```

Comment:
@casey1173 I added a new option to set focus if quill steals it... let me know what you think

Reply:
Good solution!! Glad you were able to find the source of the problem with these forms